Qt.py

```diff
@@ -43,6 +43,22 @@ def _pyqt5():
 
 
 def _pyqt4():
+    # Attempt to set sip API v2 (must be done prior to importing PyQt4)
+    import sip
+    try:
+        sip.setapi("QString", 2)
+        sip.setapi("QVariant", 2)
+        sip.setapi("QDate", 2)
+        sip.setapi("QDateTime", 2)
+        sip.setapi("QTextStream", 2)
+        sip.setapi("QTime", 2)
+        sip.setapi("QUrl", 2)
+    except AttributeError:
```

Comment:
If these functions raise anything but `ImportError`, it will throw the actual exception to the user, when he *imports* the module. This I think might be unexpected.

Rather, I think they should all throw an import error, without message, since it is discarded [here](https://github.com/fredrikaverpil/Qt.py/blob/07d9720b7f3672c27fd51f7a0d8621609c5f5660/Qt.py#L220) anyway.

If we want to throw a message, I would print it to `sys.stderr.write()`.